connectome_interpreter/compress_paths.py

```diff
@@ -78,11 +132,86 @@ def compress_paths(inprop, step_number, threshold=0, output_threshold=1e-4, root
                 out_csc.eliminate_zeros()
 
             steps_fast.append(out_csc)
-            torch.cuda.empty_cache()
+
+    # remove all variables
+    del out_tensor, inprop_tensor
+    torch.cuda.empty_cache()
 
     return steps_fast
 
 
+# # below: not chunked version
+# def compress_paths(inprop, step_number, threshold=0, output_threshold=1e-4, root=False):
```

Comment:
Uncomment and rename. Commented-out code tends to rot, because your linters and type checkers will ignore it.

If this function is meant to be _mostly_ the same as the preceding function, then you could phrase the docstring as something like:
```python
def compress_paths_without_chunking(...):
    """As above, but without chunking, and instead...
    """
```
highlighting the difference.